src/components/VisualizationScreen.tsx

```diff
@@ -10,10 +10,11 @@ const bifrostWidgetCss = css`
   // Element-based styles
   //===========================================================
   display: grid;
-  grid-template-columns: 675px minmax(450px, 600px);
+  grid-template-columns: 1fr minmax(450px, 600px);
```

Comment:
We probably want to keep a fixed value here that is wide enough to accommodate the color legend. If the sidebar and the graph get too far apart, its hard to interact between the two.